examples/remix-auth/github/.env.example

```diff
@@ -0,0 +1,3 @@
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+BASE_URL=http://localhost:3000/
```

Comment:
I'd prefer that this be derived from the request at runtime like so: https://github.com/kentcdodds/kentcdodds.com/blob/main/app/utils/misc.tsx#L229

Reply:
That should need to be supported by the strategy code, this needs to be initialized outside the loader